src/components/data/WdsTroubleshooter.ts

```diff
@@ -128,7 +129,13 @@ export const WdsTroubleshooter = ({ onDismiss, workspaceId, mrgId }) => {
 
   if (cloneSourceWorkspaceId !== null) {
     troubleShooterText.push(
-      ['Data table clone source', cloneSourceWorkspaceId, false, cloneSourceWorkspaceId !== 'unknown'],
+      [
```

Comment:
How do you know the `cloneSourceWorkspaceId`? I'm curious because it's not something we maintain on cloned workspaces, unfortunately.

Reply:
Its pulled from WDS's clone endpoint.
https://github.com/DataBiosphere/terra-ui/blob/0ae12ef9c9e049fdf26fa64404adcc7565a9b329/src/libs/ajax/WorkspaceDataService.ts#L95-L98

Reply:
Does WDS happen to already have that information? I'm wondering if you could avoid another Ajax call to retrieve it by just passing the information via the clone status.

Reply:
WDS' clone endpoint only returns the source workspace ID and status.
https://github.com/DataBiosphere/terra-workspace-data-service/blob/b479672d36545d0b45766b3e0e09564d077a19b8/service/src/main/resources/static/swagger/openapi-docs.yaml#L922-L931

Reply:
Right, I was suggesting that perhaps that endpoint could be updated to pass the additional information (assuming it already has it, which I have to think it does). That way terra-ui doesn't have to make an extra ajax call to fetch it.

Reply:
I don't know... I feel like Terra UI should be able to handle workspace IDs and we shouldn't have to pass around two different identifiers all the time. Not saying this PR is a good solution... I'd rather have Terra UI be able to handle navigating to a workspace by ID instead of requiring namespace and name.

Reply:
Then again, I guess we'd still need the namespace and name for the link text.

Reply:
@davidangb said that returning the namespace and name would require WDS to make the same call to Rawls as the UI is doing in this PR and WDS is currently unaware of Rawls.